aten/src/ATen/native/Resize.cpp

```diff
@@ -21,15 +22,51 @@ Tensor& resize_cpu_(Tensor& self, IntArrayRef size) {
 // this function in native_functions.yaml
 Tensor& resize_as_sparse_(Tensor& self, const Tensor& src);
 
-Tensor& resize_as_(Tensor& self, const Tensor& the_template) {
+
+// TODO(VitalyFedyunin): Move it to HTML docs.
+//
+// Strides of the output tensor of `resize_as_` operator is defined by input
+// tensor strides and the value of memory_format argument.
+//
+// If memory_format is not defined or equals to MemoryFormat::Contiguous
```

Comment:
I don't think this is correct.

Don't we not restride if memory_format is not set and the sizes match?

Reply:
Fixed